MAFIA/Views/Controllers/Game/GameViewController.swift

```diff
@@ -170,27 +174,47 @@ extension GameViewController: UITableViewDataSource {
 extension GameViewController: UITableViewDelegate {
     
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        let playerToEliminate = playersToDisplay[indexPath.row]
-        presenter.kill(player: playerToEliminate)
-        presenter.didEndGame()
+        //        let playerToEliminate = playersToDisplay[indexPath.row]
+        //        presenter.kill(player: playerToEliminate)
```

Comment:
We should remove this commented code since we are not using it

Reply:
Done